config-service/build.gradle.kts

```diff
@@ -75,6 +76,7 @@ dependencies {
     runtimeOnly("io.netty:netty-codec-http2:4.1.68.Final")
     runtimeOnly("io.netty:netty-handler-proxy:4.1.68.Final")
   }
+  runtimeOnly(libs.kafka.protobuf.serializer)
```

Comment:
would it make sense to specify this in the generator? Seems like this leaks a concern from the generator up here.

Reply:
Moved to generator